README.md

```diff
@@ -97,11 +102,12 @@ The following procedure describes how to deploy the `ExternalDNS` Operator for A
    $ make index-image-push
    ```
 
-5. Create the `external-dns-operator` namespace:
-   ```sh
-   $ oc create ns external-dns-operator
+5. Prepare the operand namespace:
```

Comment:
```suggestion
5. Prepare the operator namespace:
```

Reply:
"external-dns" is the operand namespace, and `config/samples/extra-roles.yaml` gives the operator permission to access resources inside the operand namespace, so isn't "Prepare the operand namespace" accurate?